src/lib/search-and-replace.js

```diff
@@ -143,11 +156,41 @@ export const searchAndReplace = async (
 	const replacements = flatten( replacementsArr );
 	debug( 'Pairs: ', pairs, 'Replacements: ', replacements );
 
+	// Add a confirmation step for search-replace
+	const yes = await confirm( [
+		{
+			key: 'From',
+			value: `${ chalk.cyan( replacements[ 0 ] ) }`,
+		},
+		{
+			key: 'To',
+			value: `${ chalk.cyan( replacements[ 1 ] ) }`,
+		},
+	], 'Proceed with the following Search and Replace values?' );
+
+	// Bail if user does not wish to proceed
+	if ( ! yes ) {
+		console.log( `${ chalk.red( 'Cancelling' ) }` );
+
+		await trackEvent( 'search_replace_cancelled', { is_import: isImport, in_place: inPlace } );
+
+		process.exit();
+	}
+
 	if ( inPlace ) {
-		await confirm(
+		const approved = await confirm(
```

Comment:
Isn't this supposed to be a confirmation step for running search-replace? Is the one added is to confirm values and this one to confirm the search-replace?

If yes, I guess it would be better to have both in the same step (printing the `from:` `to:` + a message to confirm the from/to and start the search-replace)

Reply:
This confirmation prompt is if the `--in-place` flag is set. So for regular search-replace, this prompt won't show up (but the prompt displaying the `from/to` values will), but if `search-replace` and `in-place` are both set, then both prompts will show up. Hope that clarifies things!